src/index.ts

```diff
@@ -122,15 +147,18 @@ function getEmit(
     rawFilePath: string,
     filePath: string,
     instance: interfaces.TSInstance,
+    generatedFiles: { [filePath: string]: string },
     loader: interfaces.Webpack
 ) {
     // Emit Javascript
     const output = instance.languageService.getEmitOutput(filePath);
 
-    loader.clearDependencies();
+    // loader.clearDependencies();
```

Comment:
I think what is really wanted is not to clear dependencies, but to replace the original dependency on resourcePath with a dependency on rawFilePath.
My latest commit implements this while preserving any other existing dependencies.